cpp/src/arrow/flight/sql/odbc/install/mac/install_odbc_ini.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/sh
```

Comment:
Please add the Apache license here:
For example, arrow's `sh` code with license: https://github.com/apache/arrow/blob/403ba70f838dff125c46945948b544a61d12b27f/ci/scripts/install_conda.sh#L1